src/main/java/net/floodlightcontroller/core/internal/Controller.java

```diff
@@ -368,8 +368,18 @@ public void handleMessage(IOFSwitch sw, OFMessage m,
                 }
 
                 if (alwaysDecodeEth) {
-                    eth = new Ethernet();
-                    eth.deserialize(pi.getData(), 0, pi.getData().length);
+                    try {
+                        eth = new Ethernet();
+                        eth.deserialize(pi.getData(), 0, pi.getData().length);
+                    }
+                    catch (RuntimeException e) {
+                        if (log.isDebugEnabled()) {
```

Comment:
@QingWang0909 I would put this as an WARN message. It should be a fairly uncommon occurrence.